controller/lib/core/sensors.cpp

```diff
@@ -88,7 +108,21 @@ void zero_sensors() {
       get_raw_sensor_zero_reading(PressureSensors::EXHALATION_PIN);
 }
 
-void sensors_init() {
+void sensors_init(float venturiPortDiameter, float venturiChokeDiameter) {
+  if ((venturiPortDiameter > venturiChokeDiameter) &&
+      venturiChokeDiameter > 0.0f) {
+    venturiPortArea = diameterOfCircleToArea(venturiPortDiameter);
+    venturiChokeArea = diameterOfCircleToArea(venturiChokeDiameter);
+  } else {
+    //@TODO should we indicate to caller that defaults were used since
+    // parameters were invalid?
+  }
+  // Update coefficients used to be able to convert differential pressure to
```

Comment:
Nit s/used to be able to convert/used to convert

Reply:
I english good - will fix.